github-processes.md

```diff
@@ -140,3 +141,41 @@ If more than two hotfixes are required on a particular release, the team will ca
+| 3. Design                                                         | Sponsor and Product Owner                                 | What and Why, link to parent Epic	Priority, Preliminary Acceptance Criteria and outline of work scope | In this phase, preliminary acceptance criteria are documented as a foundation for further refinement.                                                                                                                                                |
+| 4. Refinement	Product Owner, Dev and QA (where applicable)        | Preliminary Acceptance Criteria and outline of work scope | Comprehensive scenarios for Acceptance Criteria, confirmed work scope, and size                       | In this phase, engineers review the desired outcomes to ensure technical feasibility, clarify doubts, and solidify their understanding. They also estimate the required effort. Following this review, the issue is deemed Ready for implementation. |
+| 5. Implementation                                                 | Dev and QA (where applicable)                             | A comprehensive set of scenarios for Acceptance Criteria and a confirmed Scope of Work.               | A Pull Request (PR) including Unit Tests that pass and manifest files for automated testing, Technical documentation                                                                                                                                 | In this phase, engineers execute the solution and review PRs. They also record a demo to demonstrate that it meets the acceptance criteria.                      |
+| 6. UAT                                                            | Product Owner and Sponsor (if applicable)                 | Working feature and Acceptance Criteria                                                               | TBC                                                                                                                                                                                                                                                  | In this phase, the Product Owner verifies that the implementation matches the design by reviewing a demonstration of the work completed.                         |
+| 7. Rollout	\Product Owner,  Sponsor and Marketing (if applicable) | Demos, Technical Documentation                            | TBC                                                                                                   | In this phase, the Product Owner ensures that the community is prepared for the upcoming change. This preparation includes updates to the change log, project announcements, revisions to documentation, marketing efforts, and more.                |
```

Comment:
@jmcook1186 Is the slash on line 181 supposed to be a "|" ?

Reply:
yup, thanks
```suggestion
| 7. Rollout	| Product Owner,  Sponsor and Marketing (if applicable) | Demos, Technical Documentation                            | TBC                                                                                                   | In this phase, the Product Owner ensures that the community is prepared for the upcoming change. This preparation includes updates to the change log, project announcements, revisions to documentation, marketing efforts, and more.                |
```